boudicca.base/common-model/src/main/kotlin/base/boudicca/model/structured/StructuredEntry.kt

```diff
@@ -5,13 +5,16 @@ import base.boudicca.keyfilters.KeyFilters
 import base.boudicca.keyfilters.KeySelector
 import base.boudicca.model.Entry
 import base.boudicca.model.structured.dsl.StructuredEntryBuilder
+import io.github.oshai.kotlinlogging.KotlinLogging
 import java.util.*
 
 /**
  * as with the [Entry] class, a StructuredEntry is simple a typealias for a Map<Key, String>
  */
 typealias StructuredEntry = Map<Key, String>
 
+private val logger = KotlinLogging.logger("base.boudicca.model.structured.StructuredEntry")
```

Comment:
do we really need to set the logger name manually? isn't that taken automatically from the classname usually?

Reply:
there is no class here where we can take the name from :D Entries + StructuredEntries are only typealiases, so we need to set the name here manually